helmfile/charts/networkpolicy-service/values.yaml

```diff
@@ -14,6 +14,23 @@ additional: {}
 #         foo: bar
 #     ingress: {}
 #     egress: {}
+
+global:
+  apiserver:
+    ips:
+      - "0.0.0.0/0"
+    port: 6443
+  scNodes:
+    ips:
+      - "0.0.0.0/0"
+  wcIngress:
+    ips:
+      - "0.0.0.0/0"
+  objectStorage:
+    ips:
+      - "0.0.0.0/0"
+    port: 443
+
 harbor:
```

Comment:
Could you clear up the stuff you removed from here aswell? :slightly_smiling_face: core etc.